packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.8+14
+
+* Fixes issue https://github.com/flutter/flutter/issues/117945z.
```

Comment:
This is a broken link. Also, the changelog should describe fixes, not just link to an issue.